specs/bep-v1.rst

```diff
@@ -62,468 +62,274 @@ another.
 
 The underlying transport protocol MUST guarantee reliable packet delivery.
 
+In this document, in diagrams and text, "bit 0" refers to the *most
+significant* bit of a word; "bit 15" is thus the least significant bit of a
+16 bit word (int16) and "bit 31" is the least significant bit of a 32 bit
+word (int32). Non protocol buffer integers are always represented in network
+byte order (i.e., big endian) and are signed unless stated otherwise, but
+when describing message lengths negative values do not make sense and the
+most significant bit MUST be zero.
+
+The protocol buffer schemas in this document are in ``proto3`` syntax. This
```

Comment:
I want the definitions in the text so they can be discussed there, but it wouldn't hurt to provide a full schema file of course.